.github/workflows/qa.yaml

```diff
@@ -191,6 +195,7 @@ jobs:
           G_DEBUG: "fatal-criticals"
         run: |
           set -eu
+          set -x
```

Comment:
I found it really useful here, because without it it's not clear which command failed.

Reply:
Yeah, I kinda agree too... As said I'd personally use it by default, but I know not everybody may agree :)

Reply:
Yeah, we don't use it by default to avoid "clutter" on the CI reports and enable it when necessary to debug failing jobs, so I think it's best to remove it to keep consistency with all the other actions.

Reply:
I placed the `set -x` so that they don't print unnecessary commands and I use `set +x ` after the commands which we want to print were executed.